tests/tests.sh

```diff
@@ -32,6 +32,18 @@ export PATH="${ROOTDIR}:${PATH}"
 
 readonly WCURL_CMD="wcurl --dry-run "
 
+check_shunit2_compatibility() {
+    output=$(sh ./tests/compatibility.sh 2>/dev/null)
+
+    if ! echo "${output}" | grep -q "OK"; then
```

Comment:
As we discussed in person, you can simplify all of this by directly calling `assertContains` in the if statement and branching on the result of that.

This might expose the issue with macos, which is currently failing for an unknown reason.